instruction/db-sql/db-sql.md

```diff
@@ -88,20 +91,28 @@ Notice that each field is followed by the `NOT NULL` clause. That means each of
 
 The `id` field is also annotated with the `AUTO_INCREMENT` keyword. This means that you don't actually provide the `id` field when you insert a row. The database will do that for you using an automatically increasing integer.
 
+### Changing Tables
+
+> [!IMPORTANT]
```

Comment:
How about we just combine the meaningful sentence with the following paragraph rather than telling them to read the following paragraph?

Simply changing a "Create Table" statement stored in the code **does not** change the database. A `CREATE TABLE` statement _only_ creates a table. If the table already exists this command will throw an error. You can also use `CREATE TABLE IF NOT EXISTS` to conditionally create the table if it doesn't exist.